### PR TITLE
Use the openspec api from the public repository

### PR DESCRIPTION
**Pull request by lepiolet**

### CODE REVIEW

File: bin/get-open-spec.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+VERSIONS="12 13 14 15 16"
+
+rm -rf api_spec
+mkdir api_spec
+for v in 12 13 14 15 16; do
```
Comment (danijeel) [inline bin/get-open-spec.sh]:
```suggestion
for v in "${VERSIONS[@]}"; do
```
i think you forgot to use the variable here

Comment (lepiolet) [reply bin/get-open-spec.sh:6]: I did not forget, simply i did not use it. Do you want me to add it?

Comment (lepiolet) [reply bin/get-open-spec.sh:6]: My idea was not to use it

**Review by danijeel — APPROVED** [review]

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -10,10 +10,38 @@ on:
...
 
 jobs:
+  get-specs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install wget tool
+        run: |
+          sudo apt-get update
+          sudo apt-get install wget
```
Comment (thomascellerier) [inline .github/workflows/docker.yml:20]: You don't need to pass `--yes` when running apt-get install commands in github actions?

File: bin/unzip-open-spec.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -ex
+mkdir api_specs
+for v in 12 13 14 15 16; do
+    unzip /tmp/openspec-$v.zip -d api_specs
+    mv api_specs/sdp-api-specification-version-$v api_specs/v$v
+    rm -rf api_specs/v$v/.github
+    rm api_specs/v$v/README.md
```
Comment (thomascellerier) [inline bin/unzip-open-spec.sh]: Another option would be to delete all the non-yaml files.

**Review by thomascellerier — APPROVED** [review]

File: .github/workflows/docker.yml (hunk 2 above)
Comment (lepiolet) [reply .github/workflows/docker.yml:20]: no it works like this

File: bin/unzip-open-spec.sh (hunk 3 above)
Comment (lepiolet) [reply bin/unzip-open-spec.sh]: yeah well that works and in the future i will use the proper release  files

**Pull request merged**